include/fmt/format.h

```diff
@@ -991,8 +991,12 @@ FMT_CONSTEXPR20 inline auto count_digits(uint64_t n) -> int {
 template <int BITS, typename UInt>
 FMT_CONSTEXPR auto count_digits(UInt n) -> int {
 #ifdef FMT_BUILTIN_CLZ
-  if (num_bits<UInt>() == 32)
-    return (FMT_BUILTIN_CLZ(static_cast<uint32_t>(n) | 1) ^ 31) / BITS + 1;
+  // Lambda avoids error: failure was caused by call of undefined function or
+  // one not declared 'constexpr' from MSVC.
+  if (!is_constant_evaluated() && num_bits<UInt>() == 32)
+    return [](UInt m) {
+      return (FMT_BUILTIN_CLZ(static_cast<uint32_t>(m) | 1) ^ 31) / BITS + 1;
+    }(n);
```

Comment:
Why is `is_constant_evaluated` alone not enough?

Reply:
You're right. This is my mistake.